src/main/java/br/com/openenade/api/AnoController.java

```diff
@@ -0,0 +1,74 @@
+package br.com.openenade.api;
+
+import java.util.Collection;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin(origins = "*")
+@RequestMapping(path = AnoController.ENDPOINT)
+public class AnoController {
+
+    public static final String ENDPOINT = "/ano";
+
+    @Autowired
+    private AnoService service;
+
+    @Autowired
+    Utils utils;
```

Comment:
Is really necessary to use Utils as an attribute? We can use Utils methods as static right?